packages/MarkdownEditor-Core.package/MarkdownTextStyler.class/instance/privateStyle..st

```diff
@@ -2,5 +2,5 @@ private
 privateStyle: aText
 
 	| text |
-	text := (MarkdownBlockParser new) parse: aText asString.
+	text := self parse: aText asText.
```

Comment:
We were confused about the types too  :D 

The `asText` in line 5 is not necessary. That's correct. 

The main reason why we kept the `aStringOrText` in the `parse:` methods is because we always test them with strings and didn't want to deal with changing all the tests. I personally think it is a good idea to change it to always use Texts and never use strings especially since we are changing the whole classes to TextStylers which don't care about strings.

I think this PR needs some more time in our next iteration since the requested changes are too big for this one.